src/defaults/checkOptions.m

```diff
@@ -1,14 +1,115 @@
 % (C) Copyright 2019 CPP BIDS SPM-pipeline developers
 
 function opt = checkOptions(opt)
```

Comment:
thank you! <3 for the commented explanations.

Reply:
Let me know if we need to add something. :-)